### DISCUSSION TITLE
转ONNX问题

### DISCUSSION THREAD

**Discussion by weilx2267** [RapidAI/RapidOrientation#13] [Q&A]
您好，打扰了，我这边按照paddlestructure指定的方式训练版面分析，获取到了推断模型，然后使用paddle2onnx进行转onnx，然后导入到本项目下，推理的时候查看了一下输入输出：
输入：['image', 'scale_factor'] 
输出：['multiclass_nms3_0.tmp_0', 'multiclass_nms3_0.tmp_2']
使用您提供的模型文件输入输出是：
输入：['image']
输出：['transpose_0.tmp_0', 'transpose_2.tmp_0', 'transpose_4.tmp_0', 'transpose_6.tmp_0', 'transpose_1.tmp_0', 'transpose_3.tmp_0', 'transpose_5.tmp_0',
 'transpose_7.tmp_0']
请问转onnx过程有其它的哪些操作，才能完全适配您这个项目？

**Comment by SWHL** [comment]
你好，请问你这里用的版面分析模型是这个[layout](https://github.com/PaddlePaddle/PaddleOCR/blob/133d67f27dc8a241d6b2e30a9f047a0fb75bebbe/ppstructure/layout/README_ch.md)链接下的模型吗？

如果是这个链接下的，可以直接替换模型即可。

如果是其他模型结果，不能直接用了，需要你这里详细描述一下用的哪个，我来看看怎么适配。
